### PR TITLE
update quick-start for en && zh

### PR DESCRIPTION
**Pull request by yJunS**
I see the website's default language is Chinese and the repository's language is English, and access the website's URL  is not found, so I replace it.

image
image

### CODE REVIEW

**Comment by rayzhou2017** [comment]
That is the way it is since it is for Chinese users. The default language is English for the global website kubesphere.io.

**Comment by Sherlock113** [comment]
The link kubesphere.com.cn is for Chinese users. The expected Chinese doc link for quickstarts is https://kubesphere.com.cn/docs/quick-start/.

**Comment by yJunS** [comment]
Yeah, but access the link HTTP status is 404 in the link kubesphere.com.cn.
image

**Comment by Sherlock113** [comment]
> Yeah, but access the link HTTP status is 404 in the link kubesphere.com.cn.
> image

Thanks for bringing this up. There is sth wrong with the relative link. Refer to my comments.

File: content/en/contribution/_index.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ section2:
       icon2: 'images/contribution/37.png'
       children:
         - content: 'Download KubeSphere'
-          link: '../../../docs/quick-start/all-in-one-on-linux/'
+          link: '../../../en/docs/quick-start/all-in-one-on-linux/'
```
Comment (Sherlock113) [inline content/en/contribution/_index.md:18]: Change the path to /docs/quick-start/all-in-one-on-linux/

File: content/en/contribution/_index.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ section2:
...
         - content: 'Quickstart'
-          link: '../../../docs/quick-start/create-workspace-and-project/'
+          link: '../../../en/docs/quick-start/create-workspace-and-project/'
```
Comment (Sherlock113) [inline content/en/contribution/_index.md:20]: Change the path to /docs/quick-start/create-workspace-and-project/

File: content/zh/contribution/_index.md [hunk 3]
```diff
@@ -15,9 +15,9 @@ section2:
       icon2: 'images/contribution/37.png'
       children:
         - content: '下载 KubeSphere'
-          link: '../../../zh/docs/quick-start/all-in-one-on-linux/'
+          link: '../../../docs/quick-start/all-in-one-on-linux/'
```
Comment (Sherlock113) [inline content/zh/contribution/_index.md:18]: Change the path to /docs/quick-start/all-in-one-on-linux/

File: content/zh/contribution/_index.md [hunk 4]
```diff
@@ -15,9 +15,9 @@ section2:
...
         - content: '快速入门'
-          link: '../../../zh/docs/quick-start/create-workspace-and-project/'
+          link: '../../../docs/quick-start/create-workspace-and-project/'
```
Comment (Sherlock113) [inline content/zh/contribution/_index.md:20]: Change the path to /docs/quick-start/create-workspace-and-project/

**Comment by yJunS** [comment]
> > Yeah, but access the link HTTP status is 404 in the link kubesphere.com.cn.
> > image
> 
> Thanks for bringing this up. There is sth wrong with the relative link. Refer to my comments.

Already changed

**Pull request merged**